nimble/controller/src/ble_ll_sched.c

```diff
@@ -1198,7 +1206,14 @@ ble_ll_sched_run(void *arg)
         /* Remove schedule item and execute the callback */
         TAILQ_REMOVE(&g_ble_ll_sched_q, sch, link);
         sch->enqueued = 0;
-        ble_ll_sched_execute_item(sch);
+        rc = ble_ll_sched_execute_item(sch);
```

Comment:
Should `ble_ll_sched_execute_item` be called *after* updating current slot bounds below?